apps/glusterfs/app.go

```diff
@@ -98,7 +114,20 @@ func NewApp(configIo io.Reader) *App {
 		logger.Warning("Unable to open database.  Retrying using read only mode")
 
 		// Try opening as read-only
-		app.db, err = bolt.Open(dbfilename, 0666, &bolt.Options{
+		app.db, err = bolt.Open(dbfilename, 0600, &bolt.Options{
```

Comment:
The mode of the file is used for creation of the db file. It does decide with what permissions it will open it in.